roxie/ccd/ccdstate.cpp

```diff
@@ -3068,6 +3068,7 @@ class CRoxiePackageSetManager : implements IRoxieQueryPackageManagerSet, impleme
             unknown = true;
             break;
         }
+        currentTopologyHash = getTopologyHash();
```

Comment:
Should this only be done if topology has changed?  Does the topology ever get large enough that the time to generate will be significant.

Reply:
It's not huge, and this only happens on a control: message. I doubt it is detectable, but I don't mind changing if you want me to.

Reply:
I doubt it is matters.  Leave as it is.